src/Microsoft.Diagnostics.Runtime/Interfaces/IClrField.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+
+namespace Microsoft.Diagnostics.Runtime.Interfaces
+{
+    public interface IClrField
+    {
+        FieldAttributes Attributes { get; }
```

Comment:
Is FieldAttributes an enum/bit field or something else? Doesn't appear to be in 2.3.xxx

Reply:
`FieldAttributes` are the standard System.Reflection enum that backs these field/type attributes.  This is instead of `IsPublic`, `IsPrivate` etc.

It's unrelated to the interface changes here.